app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/io_models/GameGenieRequest.kt

```diff
@@ -1,11 +1,18 @@
 package com.aptoide.android.aptoidegames.gamegenie.io_models
 
 import androidx.annotation.Keep
-import com.aptoide.android.aptoidegames.gamegenie.domain.ChatInteraction
+import com.aptoide.android.aptoidegames.gamegenie.domain.ChatInteractionHistory
+import com.aptoide.android.aptoidegames.gamegenie.domain.GameContext
 
 @Keep
 data class GameGenieRequest(
   val id: String?,
   val title: String?,
-  val conversation: List<ChatInteraction>,
+  val conversation: List<ChatInteractionHistory>,
+  val metadata: GameGenieMetadata?,
```

Comment:
Does it make sense to send all installed apps on all messages? Shouldn't we be making like a sync on app startup for example ?